cmd/daemon/controller.go

```diff
@@ -38,16 +39,24 @@ var (
 	}
 	kubeconfig     string
 	windowsService bool
+	logDir         string
 )
 
 func init() {
 	rootCmd.AddCommand(controllerCmd)
 	controllerCmd.PersistentFlags().StringVar(&kubeconfig, "kubeconfig", "", "Path to kubeconfig")
+	controllerCmd.PersistentFlags().StringVar(&logDir, "log-dir", "", "Directory to write logs to")
```

Comment:
Can we add some default value here?

Reply:
Setting the default value to an empty string is standard as it defaults to log to stdout/stderr.